src/Components.tsx

```diff
@@ -18,26 +18,21 @@ export const ConnectAccountOnboarding = ({
   const {wrapper, component: onboarding} =
     useCreateComponent('account-onboarding');
 
-  useAttachAttribute(
+  React.useEffect(() => {
+    if (!onboarding) return;
+    onboarding.setRecipientTermsOfServiceUrl(recipientTermsOfServiceUrl);
```

Comment:
It is a bit odd to call all of the setters. Does it make sense to instead have separate useEffects like we had before? We can have a hook wrappers that do that as well, like:

```
useUpdateWithSetter(onboarding, fullTermsOfServiceUrl, (elem, val) => elem.setFullTermsOfServiceUrl(val));
```